docker/client.Dockerfile

```diff
@@ -7,18 +7,18 @@ RUN npm i -g npm@7.20.0
 WORKDIR /app
 
 COPY ./package.json ./package.json
+COPY ./build-plugins.js ./build-plugins.js
+COPY ./plugins/ ./plugins/
 
 # Fix for heap limit allocation issue
 ENV NODE_OPTIONS="--max-old-space-size=4096"
 
+ENV NODE_ENV=production
+
 # Build plugins
-COPY ./plugins/package.json ./plugins/package-lock.json ./plugins/
-RUN npm --prefix plugins install
-COPY ./plugins/ ./plugins/
 RUN npm run build:plugins
```

Comment:
Is this line required?

Reply:
https://github.com/ToolJet/ToolJet/pull/3024/files#diff-81041e84b391055b6814e5bca2e5c0883f825e4dd4011f987de3d2354ccbec21R11
We are copying entire plugins directory, no need to pick package and package-lock separately